### DISCUSSION TITLE
Which autogen is the official

### DISCUSSION THREAD

**Discussion by huangyingting** [microsoft/autogen#4216] [General]
I just discovered that autogen is splitted into two repos, both claims they are autogen, so which one is the offical
https://ag2ai.github.io/ag2/

**Comment by jackgerrits** [comment]
https://github.com/microsoft/autogen is the original, and official, AutoGen repo. AG2 is a fork of AutoGen 0.2 and we are not involved with it. For more details see [this statement](https://github.com/microsoft/autogen/discussions/4217)

<img width="896" alt="image" src="https://github.com/user-attachments/assets/114a1f23-c5f0-4e0f-a7cd-33ec96c1a896">

---

EDIT: Please also see our further clarification below: https://github.com/microsoft/autogen/discussions/4216#discussioncomment-11275105

<img width="892" alt="image" src="https://github.com/user-attachments/assets/eb06bdec-5ce8-46ca-a2d9-3d958c035b3d">

**Comment by qingyun-wu** [comment]
@huangyingting Thank you for raising this question!

The founding team of the project (including myself and my students from Penn State University, Chi Wang and many others) have decided to continue building AutoGen (now AG2) as part of another organization (https://github.com/ag2ai) to enable open-governance from different organizations. We believe this will best facilitate open collaboration and healthy community-led growth of the project in the long run.

AG2 will continue to serve AutoGen users via the original PyPI package `pyautogen` (and its aliases `autogen` and `ag2` ) by releasing AutoGen from AG2 (https://github.com/ag2ai/ag2)

We welcome your participation as a maintainer, contributor, or user! We're happy to discuss, answer questions, and provide 1-on-1 meetings on our Discord: https://discord.gg/pAbnFJrkgZ, which is a hub for over 20k+ active AutoGen builders!

**Reply by sonichi** [reply]
History of AutoGen:
- Initially developed by @qingyun-wu and me inside FLAML OSS, which @qingyun-wu and I started 5 years ago. @qingyun-wu is a professor from PSU when AutoGen was written.
- Spinned off from FLAML to a standalone repo in Oct 2023. @qingyun-wu and I were co-admin of the repo until recently.
- Evolved into AG2 in Nov 2024. @qingyun-wu and I are working in the AG2 repo together with the OSS contributors and we have a home on Discord https://discord.gg/pAbnFJrkgZ

**Comment by fniedtner** [comment]
@qingyun-wu and @sonichi, we appreciate your contributions to AutoGen’s success.

AutoGen’s success, evolution, and maintenance since its beginning has been shaped by efforts of many with hundreds of contributors to the code base and more than 20 Microsoft folks dedicating all or some of their time to it over the last year.

We are committed to developing and supporting the project in open-source and keep it open to all contributors. We recognize there has been some unfortunate confusion about forks, changes in access to pypi packages, and blocked permission for participation on Discord for many of our team members. See [Clarification Regarding AutoGen Forks · microsoft/autogen · Discussion #4217](https://github.com/microsoft/autogen/discussions/4217) .

The AutoGen team remains committed to the mission of AutoGen and looks forward to continuing our collaboration with the open-source community. We are excited to very soon release AutoGen 0.4, and new innovations with Magentic-1, AutoGen Studio and more. We remain excited and committed to building game-changing technology for the future of agentic technology with the open-source community.

On behalf of the AutoGen contributors at Microsoft
